examples/with-federated-passkeys/src/pages/index.tsx

```diff
@@ -150,6 +167,17 @@ export default function Home() {
               value="Create new sub-organization"
             />
           </form>
+          <div></div>
+          <h2 className={styles.prompt}>
+            OR already created a sub-org? Login!
+          </h2>
+          <form className={styles.form} onSubmit={loginFormSubmit(login)}>
```

Comment:
Maybe an abuse of the form element, but this was the most obvious way for me (a helpless frontend dev) to create a button